packages/@okta/vuepress-site/docs/reference/okta-expression-language/index.md

```diff
@@ -400,6 +400,14 @@ In [API Access Management](/docs/concepts/api-access-management/) custom authori
 To include a granted scope array and convert it to a space-delimited string, use the following expression:<br>
 `String.replace(Arrays.toCsvString(access.scope),","," ")`
 
+## Expressions in group rules
+
+The set of expressions allowed in group rules is limited to `String`, `Arrays`, and `user` expressions.
+
+For example, `user.hasBadge`, `String.stringContains(user.email, "@example.com")` and
```

Comment:
I think these might be best presented as lists. It also makes the parallelism a bit easier to see:
```suggestion
The following examples are allowed in group rule expressions:

* `user.hasBadge`
* `String.stringContains(user.email, "@example.com")`
* `Arrays.contains(user.favoriteColors, "blue")`

The following example isn't allowed in group rule expressions:

`Convert.toInt("2018") == user.yearJoined`
```

Reply:
Updated - let me know what you think.